include/aos/common/config.hpp

```diff
@@ -352,6 +352,13 @@
 #define AOS_CONFIG_PKCS11_SLOT_LIST_SIZE 5
 #endif
 
+/**
+ * Flag to choose between static/dynamic pkcs11 library
+ */
+#if defined(__ZEPHYR__) && defined(CONFIG_POSIX_API)
```

Comment:
It should be configured explicitly and set to 0 by default:
```c++
#ifndef AOS_CONFIG_PKCS11_USE_STATIC_LIB
#define AOS_CONFIG_PKCS11_USE_STATIC_LIB 0
#endif
```

Reply:
done